docs/okeps/okep-5094-layer2-transit-router.md

```
@@ -0,0 +1,894 @@
+
+For IPv6 two main things need to be done
+- Send a unsolicited RA to configure the ovn_cluster_router gateway
+- Send one unsolicited RA per node with lifetime 0 to remove the gateways from GRs.
+
+#### Upgrade cleanups
+
+The implementation should handle cleaning up the ports, routes, policies and NAT that are no longer
+configured on the gateway router or layer2 switch and remove them so there is no stale
+configuration from the previous.
+
+#### Upgrade cleanups
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)